### PR TITLE
Add documentation for kubernetes/helm

### PR DESCRIPTION
**Pull request by jlpedrosa**
Repeat PR closed by accident. @Koenkk

### CODE REVIEW

**Comment by jlpedrosa** [comment]
Went through the commits that you did on the PR and apply them again.

File: docs/guide/installation/08_kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+By default, storage is not enabled, which is great for testing.
+If you just want to use dynamic volume provisioning, just configure the `statefulset.storage.storageClassName`. 
+Persisting the volumes across installations, or you want to reuse existing data, you can leverage the options
+in `statefulset.storage` either providing a `volumeName` or selectors for volumes. Those fields get injected
+into the statefulSet persistentVolumeClaim which can link to existing volumes. 
+
+## helm
+To install the chart manually using helm, first you need to add your the helm repository to your local helm installation:
+```bash
+helm repo add zigbee2mqtt  https://charts.zigbee2mqtt.io/index
```
Comment (Joibel) [inline docs/guide/installation/08_kubernetes.md:25]: This should be `helm repo add zigbee2mqtt  https://charts.zigbee2mqtt.io/` (lose the trailing index)

**Comment by Koenkk** [comment]
I somehow missed this PR, @jlpedrosa is still up-to-date/ready for merge?

**Comment by jlpedrosa** [comment]
I can rebase this later, NP.

**Comment by jlpedrosa** [comment]
@Koenkk this is rebased, I think we should merge it.

**Comment by Koenkk** [comment]
Yes, thanks!

**Pull request merged**